plugins/katello/3.4/upgrade/smart_proxy.md

```diff
@@ -69,7 +69,7 @@ The installer with the --upgrade flag will run the right database migrations for
 {% highlight bash %}
 foreman-installer --scenario foreman-proxy-content --upgrade\
                   --foreman-proxy-content-certs-tar ~/myproxy.example.com-certs.tar\
-                  --certs-update-all --certs-regenerate --certs-deploy
+                  --certs-update-all --certs-regenerate true --certs-deploy true
```

Comment:
I think its because certs-update-all is added here https://github.com/Katello/katello-installer/blob/master/hooks/boot/20-certs_update.rb and the other two are options from puppet-certs module, so I'm not sure if its a bug or if its supposed to be like this in kafo; in any case I just noticed its not working like it says it should work in docs ;)